internal/component/database_observability/mysql/collector/setup_consumers.go

```diff
@@ -78,7 +78,7 @@ func (c *SetupConsumers) Start(ctx context.Context) error {
 
 		for {
 			if err := c.getSetupConsumers(c.ctx); err != nil {
-				level.Error(c.logger).Log("msg", "setupConsumers collector error", "err", err)
+				level.Error(c.logger).Log("msg", "collector error", "err", err)
```

Comment:
nit: while at it, can we change the message to something more specific?

Reply:
You mean in all collectors or just this one? All collectors are following this pattern.
Do you have a suggestion for a more specific message?